scripts/eval_calc_errors.py

```diff
@@ -155,6 +159,19 @@
         p["datasets_path"], dataset, split, split_type
     )
 
+    if p["eval_modality"] is None:
+        p["eval_modality"] = dp_split["eval_modality"]
+
+    if p["eval_modality"] is None:
+        scene_gt_tpath = "scene_gt_tpath"
+        scene_gt_info_tpath = "scene_gt_info_tpath"
+        scene_camera_tpath = "scene_camera_tpath"
+    else:
+        scene_gt_tpath = "scene_gt_{}_tpath".format(p["eval_modality"])
+        scene_gt_info_tpath = "scene_gt_info_{}_tpath".format(p["eval_modality"])
+        scene_camera_tpath = "scene_camera_{}_tpath".format(p["eval_modality"])
```

Comment:
Could we do this in `dataset_params.py`? It's repeated in `eval_calc_scores.py`

Reply:
not high priority though

Reply:
This was refactored